e2e/wdio/platform/pages/multi-input.po.ts

```diff
@@ -0,0 +1,54 @@
+import { BaseComponentPo } from './base-component.po';
+import { click, pause, scrollIntoView, sendKeys, waitForElDisplayed, waitForPresent } from '../../driver/wdio';
+
+export class MultiInputPo extends BaseComponentPo {
+    private url = '/multi-input';
+    root = '#page-content';
+    expandedDropdown = '.fd-list'
+    activeDropDownButtons = '[ng-reflect-disabled="false"] button[ng-reflect-glyph="value-help"]'
```

Comment:
for one you use 'Drop**d**own' and another 'Drop**D**own', let's pick 1 variant. I think 'Dropdown' is ok